<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>高空作业预警监控系统 V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/img/aqjk.png</normaloff>:/img/aqjk.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_laststaff">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family : 黑体;
font-size : 20px;</string>
           </property>
           <property name="text">
            <string>上一个</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nextstaff">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family : 黑体;
font-size : 20px;</string>
           </property>
           <property name="text">
            <string>下一个</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_all_clear">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family : 黑体;
font-size : 18px;</string>
           </property>
           <property name="text">
            <string>手动解除警报</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_notify_party">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family : 黑体;
font-size : 18px;</string>
           </property>
           <property name="text">
            <string>通知当事人</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>搜索...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchButton">
           <property name="styleSheet">
            <string notr="true">image: url(:/img/ss.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="add_facility">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>添加装备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bind_facility">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>绑定装备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rebind_facility">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>解绑装备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="del_facility">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>删除装备</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
